From 2723219c08726efa08a6bad04ffb775f850a96bc Mon Sep 17 00:00:00 2001
From: Aaron Conole <aconole@redhat.com>
Date: Mon, 3 Aug 2020 15:23:28 -0400
Subject: [PATCH] ecp22: make enum a type rather than instance

The enum defined in the qbg header is setup as a discreet instance
rather than a type.  Fix this.

Signed-off-by: Aaron Conole <aconole@redhat.com>
[Retrieved from:
https://github.com/intel/openlldp/pull/55/commits/2723219c08726efa08a6bad04ffb775f850a96bc]
Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
---
 include/qbg_ecp22.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/qbg_ecp22.h b/include/qbg_ecp22.h
index 567f6df..fe66bb3 100644
--- a/include/qbg_ecp22.h
+++ b/include/qbg_ecp22.h
@@ -49,10 +49,10 @@ enum {					/* ECP Transmit states */
 	ECP22_TX_ERROR
 };
 
-enum {
+enum ecp22_mode {
 	ECP22_REQUEST = 0,
 	ECP22_ACK
-} ecp22_mode;
+};
 
 struct ecp22_hdr {		/* ECP22 header */
 	u16 ver_op_sub;		/* ECP22 version, operation, subtype */
